commcare_connect/templates/tailwind/base_table.html

```diff
@@ -1,113 +1,165 @@
 {% load django_tables2 %}
-{% load i18n l10n %}
+{% load i18n %}
 {% block table-wrapper %}
-<div class="overflow-x-auto">
-    {% block table %}
-        <table {% render_attrs table.attrs class="min-w-full divide-y divide-gray-200" %}>
-            {% block table.thead %}
-            {% if table.show_header %}
-                <thead {{ table.attrs.thead.as_html }} class="bg-gray-50">
-                <tr>
-                {% for column in table.columns %}
-                    <th {{ column.attrs.th.as_html }} scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">
-                        {% if column.orderable %}
-                            <a href="{% querystring table.prefixed_order_by_field=column.order_by_alias.next %}">{{ column.header }}</a>
-                        {% else %}
-                            {{ column.header }}
-                        {% endif %}
-                    </th>
-                {% endfor %}
-                </tr>
-                </thead>
+<div class="scrollable-table-wrapper">
+  {% block table %}
+  <div class="scrollable-table-container">
+    <table class="scrollable-table">
```

Comment:
You have removed `render_attrs` so this template won't include the attrs you specify the python table. This will break django-tables code.